### ISSUE TITLE
Tag should not be recommanded for multi-stage

### ISSUE THREAD

**Issue by waghanza** [hadolint/hadolint#437]
Hi,

- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

Avoid `DL3006` recommandation for the last taget on multi-stage build.

### Actual behavior

`hadolint` recommand to use a tag for all images used in a container definition. As show [here](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds) we do not need to specify a tag for last image (generally `alpine` or `scratch`)

### Steps to reproduce the behavior

The following manifest
```dockerfile
FROM rust:1.44 AS build
...
FROM alpine
COPY --from=build ...
CMD ....
```
will produce a **DL3006** warning

Adding `# hadolint ignore=DL3006` before the line `FROM alpine`

I'm using `hadollint` **v1.17.7-0**, but the service in run as a github wook using http://codefactor.io

Regards,

**Comment by lorenzo** [comment]
`alpine` has many versions, and they change over time. Why is it ok not to pin the version?

**Comment by waghanza** [comment]
The main question, I think, why ?

In fact, in https://github.com/the-benchmarker/web-frameworks the purpose is to test frameworks, no matter what target distro (at least linux).

Also, in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds docker has not tagged alpine (I imagine to use **latest**)

**Comment by lorenzo** [comment]
The rationale is here: https://github.com/hadolint/hadolint/wiki/DL3006

**Comment by waghanza** [comment]
The rationale link https://docs.docker.com/engine/userguide/dockerimages/ is a redirect

However, I'm totally up with 
> You can never rely that the latest tags is a specific version.

On the contrary, my need is **NOT** to pin (the target is not a production app). Perhaps, my needs are out of the scope of `hadolint`.

Adding a magical comment do the trick

**Comment by lorenzo** [comment]
I think it is quite valid if you have different needs. You can ignore the rule by adding a comment on top of the line
